main.go

```diff
@@ -106,25 +80,71 @@ func main() {
 	default:
 		system.ErrorExit("Wrong saptune version in file '/etc/sysconfig/saptune': %s", SaptuneVersion)
 	}
+}
 
-	solutionSelector := system.GetSolutionSelector()
-	archSolutions, exist := solution.AllSolutions[solutionSelector]
-	system.AddGap(os.Stdout)
-	if !exist {
-		system.ErrorExit("The system architecture (%s) is not supported.", solutionSelector)
-		return
+func systemSetup() {
+	// only one instance of saptune should run check and set saptune lock file
```

Comment:
The comment now lost a bit of it's readability. It was split into 2 lines by intention.
So add an `.` after **run** (and before **check**) or split it again into 2 lines.